include/geomag.hpp

```diff
@@ -0,0 +1,148 @@
+// geomag2015v2.hpp Generated by python script wmmcodeupdate.py
+/** \file
+ * \author Nathan Zimmerberg (nhz2@cornell.edu)
+ * \date 29 SEP 2019
+ * \brief Header only library to calculate the magnetic field in the International Terrestrial Reference System(ITRS).
+ * \details Designed to minimize ram usage for embedded systems.
+
+The data from WMM2015, is not subject to copyright protection.
+Modifications are:
+  using ITRS coordinates,
+  conversion from nT to T,
+  Using unnormalized coefficents genrated by the python script wmmcofparsing.py
+  using a different spherical harmonics calculation, described in sections 3.2.4 and 3.2.5:
+    Satellite Orbits Models, Methods and Applications,
+      by Oliver Montenbruck and Eberhard Gill 2000
+*/
+#pragma once
```

Comment:
We should try to avoid #pragma once. I know it's clean, but it's not standard code practice yet. No telling where it breaks a CI sometime.

That is to say, it might be _pragmatic_, but unfortunately still unsupported in many areas.